API/DeleteContact.php

```diff
@@ -0,0 +1,49 @@
+<?php
+	$inData = getRequestInfo();
+	
+	$ID = $inData["ID"];
+    $firstName = $inData["firstName"];
+    $lastName = $inData["lastName"];
+    $email = $inData["email"];
+    $phone = $inData["phone"];
+    $userID = $inData["userID"];
+
+    //connection to database
+	$servername = "localhost";
+    $username = "username";
+    $password = "password";
+    $dbname = "myDB";
+
+    // Create connection
+    $conn = new mysqli($servername, $username, $password, $dbname);
+	if ($conn->connect_error) 
+	{
+		returnWithError( $conn->connect_error );
+	} 
+	else
+	{
+		//determine which piece of data that we are using to determine which contact to delete
+		$stmt = $conn->query("DELETE FROM Contacts WHERE ID='$ID', firstName='$firstName', lastName='$lastName', userID='$userID'");
```

Comment:
For deletion of a contact, the only thing you need is $ID... All contacts have unique $ID which means, all other parameters are unneeded 
```suggestion
		$stmt = $conn->query("DELETE FROM Contacts WHERE ID='$ID'");
```